code/datums/mutations/body.dm

```diff
@@ -169,11 +169,11 @@
 	REMOVE_TRAIT(owner, TRAIT_DEAF, GENETIC_MUTATION)
 
 
-//Monified turns you into a monkey.
+//Primal turns you into a lesser form of your species, if your species has that.
 /datum/mutation/human/race
-	name = "Monkified"
-	desc = "A strange genome, believing to be what differentiates monkeys from humans."
-	text_gain_indication = "You feel unusually monkey-like."
+	name = "Primal"
+	desc = "A strange genome, holding a \"genetic memory\" of the primal ancestors of this species. In case of activation, a backward evolution process will trigger."
```

Comment:
does this mean that the monkeys from genetics were actually originally humans who were forcibly turned into monkeys?